docs/mutect/mutect.tex

```diff
@@ -161,45 +161,41 @@ \section{Strand Artifact Model}
 \begin{figure}
 \centering
 \includegraphics[width=0.3\textwidth]{strand_artifact_pgm.png}
-\caption{\label{fig:strand artifact}The probabilistic graphical model for the Strand Artifact model}
+\caption{\label{fig:strand artifact}The probabilistic graphical model for the strand artifact model}
 \end{figure}
 
-The strand artifact filter detects sequencing artifact in which the evidence for the alt allele consists entirely of forward reads alone or reverse reads alone. We must detect this while taking into account the fact that at some loci, such as near the end of an exome target, \emph{all} reads are biased towards one direction, and thus the bias towards one read direction among the alt is not suspect.
+The strand artifact filter detects sequencing artifacts in which the evidence for the alt allele consists entirely of forward strand reads alone or reverse strand reads alone. We must detect this while taking into account the fact that at some loci, such as near the end of an exome target, \emph{all} reads are biased towards one direction, and therefore a bias towards a particular strand among alt is no cause for alarm.
 
-Let $z$ be a discrete latent random variable that represents the artifact state of a suspected variant, where $z \in \{ \text{Artifact Forward}, \text{Artifact Reverse}, \text{No Artifact} \}$. At each locus, the distributions over forward alt reads $x^+$ and reverse alt reads $x^-$, conditional on the artifact state $z$ as well as the allele fraction $f$ and artifact allele fraction $\epsilon$, are binomial random variables . Artifactual allele fraction $\epsilon$ is the probability that we mistakenly read a ref allele as alt due to a strand artifact error. We give the flat prior over $f$ and a beta prior over $\epsilon$. We define the conditional probability of $x^+$ as
+Let $z \in \{ z_+, z_-, z_o \}$ be a latent random variable with 1-hot encoding that represents the artifact state of a suspected variant. $z_+ = 1$ when the candidate variant is in fact a forward strand artifact, $z_- = 1$ when it's a reverse artifact, and $z_o = 1$ when there's no artifacts. At each locus, the conditional distribution over the number of forward alt reads $x^+$ is a binomial random variable
```

Comment:
You don't need the "in fact."  "There's no artifacts" --> "there is no artifact"